arch/risc-v/src/mpfs/mpfs_ihc.c

```diff
@@ -105,6 +105,8 @@
 #define VRING1_NOTIFYID          3                                  /* Vring1 id                  */
 #endif
 
+#define VQID_SHIFT               0
```

Comment:
just drop the define altogether here; we are not going back to having the shift are we?

Reply:
Decided to drop the define as discussed. Will do it also in tc_linux side.